projects/tuist/fixtures/multiplatform_app_with_sdk/Project.swift

```diff
@@ -0,0 +1,57 @@
+import ProjectDescription
```

Comment:
👍 This really helpful for testing


is it worth expanding this fixture to be `multiplatform_app`  to also show case different `destinations` for a single target?